dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -294,6 +294,7 @@ def test_sample_size_warning_in_the_profiler(self, *mocks):
         # structure data profile mock
         sdp_mock = mock.Mock()
         sdp_mock.clean_data_and_get_base_stats.return_value = (None, None)
+        sdp_mock.sample_size = 3
```

Comment:
why is this necessary?

Reply:
This test case kept throwing mock errors after I added some logic to update_from_chunk, because when I tried to calculate `self._complete_rows_sampled` for the Profile the underlying `StructuredDataProfile` was mocked out, but I still needed its sample size for a calculation.